plugins/inputs/t128_tank/README.md

```diff
@@ -0,0 +1,23 @@
+# 128T TANK Input Plugin
+
+The tank input plugin collects data from a 128T.
+
+## Configuration
+
+```toml
+[[inputs.t128_tank]]
+## A name for the collector which will be used as the measurement name of the produced data.
+# collector_name = "event_collector"
+
+## Required. A path for index file.
```

Comment:
The index file isn't required. Also, anything that has a default isn't "required".